test/pjrt/test_experimental_pjrt_tpu.py

```diff
@@ -241,16 +211,14 @@ def _reduce_scatter(pin_layout):
     )
     xm.mark_step()
 
-    return xm.get_ordinal(), out.cpu().numpy()
+    return out.cpu().numpy()
```

Comment:
why don't we need to return ` xm.get_ordinal()` here?

Reply:
`run_multiprocess` already gives the replica ordinal for each result, so we don't need to return it from this function.